web/frontend/components/HealthSummary/HealthSummary.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+const getHealthIcon = (health) => {
+  switch (health) {
+    case 'critical':
+      return <i className="eos-icons health-summary-icon text-danger">error</i>;
+    case 'warning':
+      return (
+        <i className="eos-icons health-summary-icon text-warning">warning</i>
```

Comment:
Why in the hell `prettier` has formatted so strangely hehe
Is it due the difference of line length?

Reply:
Yes it is! it looks strange the first times you see it, but it's fine :smile: